azext_iot/tests/central/json/device_template.json

```diff
@@ -7,9 +7,7 @@
     "displayName": "duplicate-field-name",
     "capabilityModel": {
         "@id": "urn:sampleApp:modelOne_bz:2",
-        "@type": [
-            "CapabilityModel"
-        ],
+        "@type": "CapabilityModel",
```

Comment:
should we move the old ones to something like a /v1 folder, and the new ones to the /v2 folder?

Reply:
Same here

Reply:
currently we support both v1 and v2 dont see additional value in separating them out as v1 and v2. V2 is a super set of